scripts/disasm

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Change this shebang (i.e., `#!/bin/bash`) to `#!/usr/bin/env bash`.
See https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash

Reply:
done.  Waiting for guidance on previous comment before I commit.